excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -222,7 +222,62 @@ public static String getCellValue(Workbook wb, Cell cell) {
             } else {
                 // TODO: Consider not formatting it, but simple using
                 // Double.toString(...)
-                result = _numberFormat.format(cell.getNumericCellValue());
+                result = getNumericCellValueAsStringUsingBuildinFormat(cell.getCellStyle(), cell.getNumericCellValue());
+            }
+            break;
+        case STRING:
+            result = cell.getRichStringCellValue().getString();
+            break;
+        default:
+            throw new IllegalStateException("Unknown cell type: " + cell.getCellType());
+        }
+
+        logger.debug("cell {} resolved to value: {}", cellCoordinate, result);
+
+        return result;
+    }
+
+    public static Object getCellValueAsObject(final Workbook wb, final Cell cell) {
```

Comment:
Can you make this method private and rename the `wb` parameter to `workbook`?

Reply:
done